api/src/Entity/Discussion.php

```diff
@@ -61,10 +72,39 @@
                 ],
             ],
         ],
-        summary: 'Get number of message not read',
-        description: 'Get number of message not read for all discussions',
+        summary: 'Get number of message not read for a repairer',
+        description: 'Get number of message not read for all discussions of a repairer',
     ),
     paginationEnabled: false,
+    security: 'is_granted("IS_AUTHENTICATED_FULLY") and user.isAssociatedWithRepairer(repairer_id)',
+)]
+#[GetCollection(
+    uriTemplate: '/customers/messages_unread',
+    controller: NumberOfMessageNotReadForCustomerController::class,
+    openapi: new Operation(
+        responses: [
+            '200' => [
+                'description' => 'Number of message not read',
+                'content' => [
+                    'application/json' => [
+                        'schema' => [
+                            'type' => 'object',
+                            'properties' => [
+                                'count' => [
+                                    'type' => 'integer',
+                                    'example' => 2,
+                                ],
+                            ],
+                        ],
+                    ],
+                ],
+            ],
+        ],
+        summary: 'Get number of message not read of a customer',
+        description: 'Get number of message not read for all discussions of a customer',
+    ),
+    paginationEnabled: false,
```

Comment:
```suggestion
    read: false,
```
to bypass the automatic retrieval of the entity, else you will have a useless request that get all discussion for the user.
```
SELECT d0_.id AS id_0, d0_.created_at AS created_at_1, d0_.last_message AS last_message_2, r1_.id AS id_3, r1_.name AS name_4, r1_.description AS description_5, r1_.mobile_phone AS mobile_phone_6, r1_.street AS street_7, r1_.street_number AS street_number_8, r1_.city AS city_9, r1_.postcode AS postcode_10, r1_.country AS country_11, r1_.latitude AS latitude_12, r1_.longitude AS longitude_13, ST_AsEWKT(r1_.gps_point) AS gps_point_14, r1_.first_slot_available AS first_slot_available_15, r1_.opening_hours AS opening_hours_16, r1_.optional_page AS optional_page_17, r1_.enabled AS enabled_18, r1_.comment AS comment_19, r1_.slug AS slug_20, r1_.created_at AS created_at_21, r1_.duration_slot AS duration_slot_22, r1_.number_of_slots AS number_of_slots_23, r1_.google_access_token AS google_access_token_24, r1_.google_refresh_token AS google_refresh_token_25, m2_.id AS id_26, m2_.file_path AS file_path_27, u3_.id AS id_28, u3_.email AS email_29, u3_.roles AS roles_30, u3_.email_confirmed AS email_confirmed_31, u3_.validation_code AS validation_code_32, u3_.forgot_password_token AS forgot_password_token_33, u3_.password AS password_34, u3_.street AS street_35, u3_.city AS city_36, u3_.last_name AS last_name_37, u3_.first_name AS first_name_38, u3_.last_connect AS last_connect_39, u3_.firebase_token AS firebase_token_40, u3_.telephone AS telephone_41, d0_.repairer_id AS repairer_id_42, d0_.customer_id AS customer_id_43, r1_.owner_id AS owner_id_44, r1_.thumbnail_id AS thumbnail_id_45, r1_.description_picture_id AS description_picture_id_46, m2_.owner_id AS owner_id_47 FROM discussion d0_ INNER JOIN repairer r1_ ON d0_.repairer_id = r1_.id LEFT JOIN media_object m2_ ON r1_.thumbnail_id = m2_.id INNER JOIN "user" u3_ ON d0_.customer_id = u3_.id ORDER BY d0_.id ASC
```
and `paginationEnabled` can be removed as it is not needed here